src/styles/ParentStyles/index.less

```diff
@@ -110,15 +110,18 @@ li {
   text-align: center;
   width: 100%;
 }
+
```

Comment:
Let's keep this consistent with the rest of the styling in this file, it looks like it's been decided to not add spaces between the selectors. If you're trying to section out a space in this file for a particular set of styles, use comments to express that and create the spacing.

Reply:
I will remove the spacing.